tests/bootstrap_spec.lua

```diff
@@ -0,0 +1,34 @@
+local a = require "plenary.async_lib.tests"
```

Comment:
Why not using `describe` and `it` directly ?

Reply:
It's `luacheck` or `sumneko` that kept complaining. do I have to add these keywords to the globals list?

Reply:
That's what I would do if possible

Reply:
I was able to get `luacheck` to recognize the library, but I could not do the same for the language server..